apps/web/src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,19 @@
+import materializeDocumentsAtCommit from '$/services/commitService/materializeDocumentsAtCommit'
+
+import DocumentTree, { CreateNode } from './DocumentTree'
+
+export default async function Sidebar() {
+  const nodes = await materializeDocumentsAtCommit('HEAD', true)
```

Comment:
Can you explain how this works? Also please use an object for the arguments. What does `true`?